jetbrains-core/tst/software/aws/toolkits/jetbrains/services/apprunner/actions/ViewLogsActionTest.kt

```diff
@@ -75,11 +75,11 @@ class ViewLogsActionTest {
             }
         }
 
-        runBlocking {
-            viewLogGroup(node, "service")
-        }
+        viewLogGroup(node, "service")
 
-        val windows = toolWindowManager.findPrefix("")
+        val windows = runInEdtAndGet {
```

Comment:
Does this work since it runs the function in the UI thread?

Reply:
Tests aren't ran on the UI thread, unless you mean the contents of the function?

Reply:
Anyway, runInEdit is smart enough to run the code block if it is already on the UI thread as if it was inlined.

Reply:
Yeah I meant the lambda itself. Was looking at the source for `runInEdtAndGet` and saw it was in-lining then making a separate call. I figured the only way this test would be stable is if the lambda was ran after UI changes propagate.

Reply:
hmmm.. I could move vieLogsGroup into the block to make sure it is ran in 1 go to be 100% sure

Reply:
Nope that seemed to break it haha.

Reply:
I broke our checkstyle, tests pass :D